Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -188,6 +188,14 @@ export default function widgetBehavior(publicAPI, model) {
   };
 
   publicAPI.translateCenterOnCurrentDirection = (nbSteps, renderer) => {
+    // As we don't select a representation actor, the update method name is
+    // not set (normally set by the ResliceCursorContextRepresentation)
+    const methodName = InteractionMethodsName.TranslateCenter;
```

Comment:
Where (what file) is InteractionMethodsName.TranslateCenter defined ?

Reply:
vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/Constants

Reply:
There are also used in ResliceCursorContextRepresentation

Reply:
(my bad, my master was not up to date)

Reply:
and why do you need to set `updateMethodName` ?

Reply:
Because if you don't click on a representation actor (line, sphere), then this method is empty. Or, we definitely apply an action related to an InteractionMethodsName.
If the user wants to know which action is currently managed, then he can use : `widget.getWidgetState().getUpdateMethodName()`. Then, he can know if we are actually rotatitin/translating axis/translating center.
It's interesting when we want to define different interaction/rendering (example: behaviors of focal point)

Reply:
Understood, if it is "interesting", that means it is not a "fix" but "feat" ?

How do you know it is a center translation and not an axis translation?

And if it is an axis translation, then you would overwrite it with center translation?

Reply:
I think it's more a feat because during interaction, this attribute should always be filled. And when we scroll, it wasn't.
And it's a fix, because if you make a rotation and then scroll, the `updateMethodName` will keep 'rotate'. Why ? because the only place it's overwrite is in the `ResliceContextRepresentation`. And if you don't pick an actor, then you never go into this class.

I know it's a center translation because I put this piece of code in `translateCenterOnCurrentDirection` method, which is only called when we scroll along slices (mouse scroll or mouse click + move).